_data/simple-icons.json

```
@@ -5497,6 +5497,11 @@
             "hex": "0076D6",
             "source": "https://compass-ssl.microsoft.com/assets/c8/67/c867db4c-f328-45b8-817c-33834c70aae6.svg?n=IE.svg"
         },
+        {
+            "title": "InterWorks",
+            "hex": "000000",
+            "source": "https://user-images.githubusercontent.com/109748036/180245879-0c831ea8-0ee3-4223-bbf1-0ae216740692.svg"
```

```suggestion
            "source": "https://interworks.com"
```

SVG icon now available at the homepage, could you please update the source, and double check your contribution matches up to this version?